app/serializers/event_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class EventSerializer < ActiveModel::Serializer
+  attributes :id, :name, :description, :image, :url, :date, :location, :instructions, :contact, :image_file_name, :image_content_type, :image_file_size, :campaign_id
```

Comment:
I think :image sends the url of the image and that should be enough for it to be used. :$